ViAn/GUI/mainwindow.cpp

```diff
@@ -155,17 +156,18 @@ void MainWindow::setup_video_player(video_player *mplayer) {
  * @param analysis
  * Slot for saving analysis to file.
  */
-void MainWindow::analysis_finished(Analysis analysis) {
+void MainWindow::analysis_finished(AnalysisMeta analysis) {
     if(current_analysis != nullptr) {
         QString text = "(done)";
         text.append(current_analysis->name);
         current_analysis->setText(0, text);
         // Save analysis
         ID p_id = ((MyQTreeWidgetItem*)current_analysis->parent()->parent())->id;
         ID v_id = ((QTreeVideoItem*)current_analysis->parent())->id;
-        analysis.name = current_analysis->name;
-        current_analysis->id = file_handler->get_project(p_id)->add_analysis(v_id, analysis);
-        file_handler->save_project(p_id);
+        analysis.m_name = current_analysis->name.toStdString();
+        std::cout << "*** analysis finished***" << analysis.m_name << std::endl;
```

Comment:
Neither this or the "done" a few lines up are a good way to indicate a finished analysis. but please capitalize them so it looks better.
a better indication for a finished analysis comes later.